src/Common/EventStore/PollableEventStore.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Gaming\Common\EventStore;
+
+use Gaming\Common\EventStore\Exception\EventStoreException;
+use Gaming\Common\EventStore\Exception\UnrecoverableException;
+use Gaming\Common\Normalizer\Exception\NormalizerException;
+
+interface PollableEventStore
+{
+    /**
+     * @return StoredEvent[]
+     * @throws UnrecoverableException
+     * @throws EventStoreException
+     * @throws NormalizerException
+     */
+    public function since(int $id, int $limit): array;
+
+    /**
+     * @throws UnrecoverableException
+     * @throws EventStoreException
+     */
+    public function hasUncommittedStoredEventId(int $id): bool;
```

Comment:
This method was already included in the "old" `EventStore` interface. It's still an implementation detail and only part of the interface because of the `ConsistentOrderPollableEventStore` decorator. Instead of using a decorator, using a strategy or a trait should fit the design better and would free the `PollableEventStore` interface from this method.